packages/good-design/src/apps/goodid/idcard/GoodIdCard.tsx

```diff
@@ -48,10 +47,38 @@ const CardRowItem = withTheme({ name: "CardRowItem" })(
 );
 
 const GoodIdCard = withTheme({ name: "GoodIdCard", skipProps: "credentialsList" })(
-  ({ credentialsList, account, isWhitelisted, avatar, fullname, expiryDate, ...props }: GoodIdCardProps) => {
+  ({ account, isWhitelisted, avatar, fullname, expiryDate, ...props }: GoodIdCardProps) => {
     const { title, subHeading, subContent, footer } = props.fontStyles ?? {};
     const truncatedAccount = truncateMiddle(account, 11);
 
+    const certificates = useAggregatedCertificates(account);
+
+    const verifiedValues = useMemo(() => {
+      let Age: string | null = null,
+        Gender: string | null = null,
+        Country: string | null = null;
+
+      const filtered = certificates?.filter(
+        certificate => certificate.typeName === "Identity" || certificate.typeName === "Location"
+      );
+
+      filtered?.forEach(({ certificate, typeName }) => {
+        const { credentialSubject } = certificate ?? {};
```

Comment:
this should be part of CardRowItem. depending of type you pick up different values from subject and display them the different way